src/client/Client.cc

```diff
@@ -423,11 +423,11 @@ int Client::init()
 
   // logger
   PerfCountersBuilder plb(cct, "client", l_c_first, l_c_last);
-  plb.add_time_avg(l_c_reply, "reply");
-  plb.add_time_avg(l_c_lat, "lat");
-  plb.add_time_avg(l_c_wrlat, "wrlat");
-  plb.add_time_avg(l_c_owrlat, "owrlat");
-  plb.add_time_avg(l_c_ordlat, "ordlat");
+  plb.add_time_avg(l_c_reply, "reply", "Latency of receiving a reply on request");
+  plb.add_time_avg(l_c_lat, "lat", "Latency of processing a request");
+  plb.add_time_avg(l_c_wrlat, "wrlat", "Latency of write operation");
+  plb.add_time_avg(l_c_owrlat, "owrlat", "Write to object latency");
+  plb.add_time_avg(l_c_ordlat, "ordlat", "Read from object latency");
```

Comment:
And these two, though present in the system, appear to be unused (except in the SyntheticClient). :(